utils/dev/run-server.sh

```diff
@@ -8,25 +8,27 @@ command -V mc
 
 projRoot=`git rev-parse --show-toplevel`
 
-dir="$1"
-if [ -z "$dir" ]; then
-	dir="/tmp/lbdev"
-fi
+cd $projRoot/server/server
+cargo check
```

Comment:
wondering if this is worthwhile, or in what circumstances you're gonna be facing 2 completely clean builds, but let's roll with it for now

Reply:
This is to solve for: build fails and mino and redis aren't terminated. I also think "does server compile" comes logically before "can minio and redis start."

Reply:
yeah I wonder if your first pass at that was better then, `cargo build @` then run that binary